pabi_budget_drilldown_report/xlsx_template/templates.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        # Budget Plan
+        <!-- Directory Budget Plan Templates -->
+        <record model="document.directory" id="dir_budget_drilldown_template">
+            <field name="name">Budget Drilldown Templates</field>
+            <field name="parent_id" ref="document.dir_root"/>
+            <field name="user_id" ref="base.user_root"/>
+            <field name="ressource_id">0</field>
+            <field name="group_ids" eval="[(4, ref('base.group_user'))]" />
+        </record>
+
+        <!-- Unit Based Template -->
+        <record id="budget_drilldown_unit_base_xlsx_template" model="ir.attachment">
```

Comment:
This is not only for template, please name it "budget_overview_report_xlsx_template"